webapp/tests/mocha/unit/purger.spec.js

```diff
@@ -64,21 +64,31 @@ describe('Purger', () => {
       });
     });
 
-    it('should throw fetch errors', () => {
```

Comment:
Could we keep this test and just add the new one where the server returns 500? Fetch throws when not connected, which is much more likely to occur frequently than API failing.

Reply:
Absolutely. Done.